# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/filters/reftest.list
# Commit: 2b42abbdb0df
# Full Hash: 2b42abbdb0df38f31dfa1178fe3b5f773f8e4812
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 9fa6f71111d0 -r 2b42abbdb0df layout/reftests/svg/filters/reftest.list
--- a/layout/reftests/svg/filters/reftest.list	Sun Feb 20 18:56:54 2022 +0000
+++ b/layout/reftests/svg/filters/reftest.list	Sun Feb 20 18:56:55 2022 +0000
@@ -86,7 +86,7 @@
 == filter-marked-line-08.svg pass.svg
 == filter-marked-line-09.svg pass.svg
 == filter-nested-filtering-01.svg pass.svg
-== filter-nested-filtering-02.svg pass.svg
+fuzzy(0-10,0-1200) == filter-nested-filtering-02.svg pass.svg
 == filter-patterned-rect-01.svg pass.svg
 == filter-patterned-rect-02.svg pass.svg
 == filter-region-01a.html pass.svg